programs/drift/src/lib.rs

```diff
@@ -161,6 +161,14 @@ pub mod drift {
         handle_place_and_make_perp_order(ctx, params, taker_order_id)
     }
 
+    pub fn place_swift_taker_order<'c: 'info, 'info>(
```

Comment:
nit: dont feel super strong about this, but i think naming thing something like "place_order_with_signed_message" is more explicit naming?